en/docs/setup/setting-up-separate-databases-for-clustering.md

```diff
@@ -4,125 +4,91 @@ This section guides you through the logical separation of data that you
 <td><code>             REGISTRY_DB            </code></td>
-<td>Shared database for config and governance registry mounts in the product's nodes. This includes data on tenants and keystores.</td>
-<td><pre><code>&lt;IS_HOME&gt;/dbscripts/</code></pre></td>
-<td><code>             &lt;IS_HOME&gt;/repository/conf/datasources/master-datasources.xml            </code></td>
+<td>
+<li>Shared database for config and governance registry mounts in the product's nodes. This includes data on tenants and 
+keystores.</li>
+<li>In the default database configuration, user data and registry data is located inside <code>SHARED_DB</code>. 
+During a deployment user data can be configured to a different db while registry data remain in the <code>SHARED_DB</code>
+.</li>
+<li>For more information, see <a href="../../administer/working-with-databases">Working with Databases</a>.</li>
+</td>
 </tr>
 <tr class="even">
-<td><code>             CONSENT_MGT            </code></td>
-<td>This is used for data pertaining to user consents.</td>
-<td><code>             &lt;IS_HOME&gt;/dbscripts/consent            </code></td>
-<td><code>             &lt;IS_HOME&gt;/repository/conf/datasources/master-datasources.xml            </code></td>
+<td><code>             CONSENT_MGT          </code></td>
+<td>
+<li>Used for data pertaining to user consents.</li>
+<li>In the default database configuration, UMA, concent and identity data are located <code>IDENTITY_DB</code>. 
+In a deplyment both identity and UMA can be seperated for a single db while concent data is configured to a seperate 
+db.</li>
+<li>For more information, see <a href="../../administer/working-with-databases">Changing the Default Datasource for 
+Consent Management Database</a>.</li>
+</td>
 </tr>
 </tbody>
 </table>
 
-To understand this concept further, see the following diagram.
-
-![Data types](../assets/img/setup/data-types.png)
-
-For more information on the concept of sharing governance and config
-registry databases across the cluster, see the topic on [Sharing
-Databases in a
-Cluster](../../administer/sharing-databases-in-a-cluster).
-
-Do the following configurations to implement the database setup.
-
-#### Creating the databases
-
-Create the databases that you require using the RDBMS of your choice and
-run the relevant script for the RDBMS of your choice according to the
-table above. For instructions on how to do this, see [Setting up the
-Physical
-Database](../../administer/setting-up-the-physical-database).
-
-#### Configuring the datasources
-
-Configure the datasources for the databases that you created above in
-both the WSO2 Identity Server nodes of your cluster. For instructions on
-how to configure the datasources for the databases you created, see
-[Changing the Carbon
-Database](../../administer/changing-the-carbon-database).
+!!! note
+    For more information on `Shared db` and `Identity db`, 
+    see [Working with Databases](#../../administer/working-with-databases/).
```

Comment:
remove hash in the relative path

Reply:
Fixed: 55e7088941e0c75d2805c48dc949bcff9db0324c